index.html

```diff
@@ -2301,9 +2299,187 @@ <h2>Sets and Lists</h2>
-  are not allowed in this version of JSON-LD. This decision was made due to the
-  extreme amount of added complexity when processing lists of lists.</p>
+<p>The implementation of <a>lists</a> in RDF depends on linking anonymous nodes
+  together using the properties <code>rdf:first</code> and
+  <code>rdf:rest</code>, with the end of the list defined as the resource
+  <code>rdf:nil</code>. This can be represented as triples, as the following
+  example shows:</p>
+
+<table class="example">
+<thead><tr>
+  <th>Subject</th>
+  <th>Property</th>
+  <th>Value</th>
+</tr></thead>
+<tbody>
+<tr>
+  <td>http://example.org/people#joebob</td>
+  <td>foaf:nick</td>
+  <td>_:b0</td>
+</tr>
+<tr>
+  <td>_:b0</td>
+  <td>rdf:first</td>
+  <td>joe</td>
+</tr>
+<tr>
+  <td>_:b0</td>
+  <td>rdf:rest</td>
+  <td>_:b1</td>
+</tr>
+<tr>
+  <td>_:b1</td>
+  <td>rdf:first</td>
+  <td>bob</td>
+</tr>
+<tr>
+  <td>_:b1</td>
+  <td>rdf:rest</td>
+  <td>_:b2</td>
+</tr>
+<tr>
+  <td>_:b2</td>
+  <td>rdf:first</td>
+  <td>jaybee</td>
+</tr>
+<tr>
+  <td>_:b2</td>
+  <td>rdf:rest</td>
+  <td>rdf:nil</td>
+</tr>
+</tbody>
+</table>
+
+<p>JSON-LD provides a syntactic shortcut for these lists. In Turtle, the graph would be expressed as follows:</p>
+
+<pre class="example" data-transform="updateExample"
+     data-content-type="text/turtle"
+     title="An ordered collection of values in Turtle">
+<!--
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+
+<http://example.org/people#joebob> foaf:nick ("joe" "bob" "jaybe") .
+-->
+</pre>
+
+<p class="changed">In JSON-LD 1.1, lists of lists, where the value of
+  a <a>list object</a>, may itself be a <a>list object</a> recusively, are
+  fully supported. For example, in <em>GeoJSON</em> (see [[RFC7946]]),
+  <em>coordinates</em> are an ordered list of <em>positions</em>, which are
+  represented as an array of two or more numbers. For example:</p>
+
+<pre class="example changed"
+     data-content-type="application/json"
+     title="Coordinates expressed in GeoJSON">
+{
+  "type": "Feature",
+  "bbox": [-10.0, -10.0, 10.0, 10.0],
+  "geometry": {
+    "type": "Polygon",
+    "coordinates": [
+        [
+            [-10.0, -10.0],
+            [10.0, -10.0],
+            [10.0, 10.0],
+            [-10.0, -10.0]
+        ]
+    ]
+  }
+  //...
+}
+</pre>
+
+<p class="changed">For this example, it's important that the values
+  expressed within coordinates maintain their order, which requires the use of
+  embedded list structures. In JSON-LD 1.1, we can express this using recursive
+  lists, by simply adding the appropriate context definion:</p>
+
+<pre class="example changed"
+     data-content-type="application/json"
+     data-transform="updateExample"
+     title="Coordinates expressed in JSON-LD">
+{
+  "@context": {
+    "@vocab": "https://purl.org/geojson/vocab#",
+    "type": "@type",
+    "bbox": {"@container": "@list"},
+    "coordinates": {"@container": "@list"}
```

Comment:
In https://github.com/geojson/geojson-ld/issues/34 we discussed an `@ndarray`, but `@list` looks just as good, or better in that it's easier to explain and understand.